packages/client/ui/react-ui/src/providers/CrossmintWalletProvider.tsx

```diff
@@ -20,13 +32,13 @@ type WalletContext = {
     status: WalletStatus;
     wallet?: EVMSmartWallet;
     error?: SmartWalletError;
-    getOrCreateWallet: (config?: WalletConfig) => { startedCreation: boolean; reason?: string };
+    getOrCreateWallet: (config?: WalletConfig) => Promise<{ startedCreation: boolean; reason?: string }>;
```

Comment:
For now, let's avoid letting clients pass in the callbacks through the react SDK.

It's unclear that's something people need, and it keeps things simpler for us.

Reply:
Do you think I should do something like 

```
getOrCreateWallet: (config?: Omit<WalletConfig, "onCreateWalletPasskeyCallback" | "onErrorCreateWalletCallback"> ) => Promise<{ startedCreation: boolean; reason?: string }>;
```